spec/integration/http.spec.js

```diff
@@ -90,13 +105,28 @@ describe( 'HTTP', function() {
 
 		harness.addResource( {
 			name: 'test',
+			middleware: [
+				function( envelope, next ) {
+					if( envelope.headers.accept ) {
+						var accept = envelope.headers.accept;
+						var match = /[.]v([0-9]*)/.exec( accept );
```

Comment:
Does this need to be pinned to the end of the Accept header? `application/hal+json.v3` vs `application.v3/hal+json`

Maybe the regex should be `/[.]v([1-9][0-9]*)$/`